library/lib/build.js

```diff
@@ -194,6 +197,16 @@ module.exports = function build({ watch }) {
         ]}]
       },
       plugins: [
+        ...watch ? [] : [
+          new webpack.LoaderOptionsPlugin({
```

Comment:
These plugins are not a pair, but should both run in production mode.
The `LoaderOptionsPlugin` is a plugin for webpack 2 -> 3 migration: https://webpack.js.org/plugins/loader-options-plugin

I'm not sure this plugin is necessary, but I have no time and don't feel like to dive in all plugins and loaders to see how they handle configuration and if they are webpack 3-ready :-)